packages/core/src/compatibility/index.ts

```diff
@@ -0,0 +1,8 @@
+/*
+ * Copyright 2017 Palantir Technologies, Inc. All rights reserved.
+ * Licensed under the BSD-3 License as modified (the “License”); you may obtain a copy
+ * of the license at https://github.com/palantir/blueprint/blob/master/LICENSE
+ * and https://github.com/palantir/blueprint/blob/master/PATENTS
+ */
+
+export * from "./browser";
```

Comment:
What do you mean? There isn't a `utils` directory. I'm more following the structure we have for accessibility - I'd expect we'll get some os specific compatibility issues at some point too.